mgmt_cli_test.py

```diff
@@ -1870,6 +1910,65 @@ def run_read_stress_and_report(self, label):
         }
         self.report_to_argus(ManagerReportType.READ, read_stress_report, label)
 
+    def run_stress_and_report(self, label):
+        """
+        Run all read and write stress in parallel.
+        Wait for all to finish.
+        Report its perf stats to Argus.
+        Args:
+            label: a postfix for reporting
+        """
+        stress_write_cmd = self.params.get('stress_cmd')
+        stress_read_cmd = self.params.get('stress_read_cmd')
+        stress_write_queue = self._run_stress_batch(cmds=stress_write_cmd)
+        stress_read_queue = self._run_stress_batch(cmds=stress_read_cmd)
+
+        def get_stress_averages(queue):
```

Comment:
Could we reuse logic from performance tests?

Reply:
fixed